src/app/containers/InlineLink/index.jsx

```diff
@@ -9,6 +8,8 @@ import fragment from '../Fragment';
 import { inlineLinkModelPropTypes } from '#models/propTypes/inlineLink';
 import { articlePath } from '../../routes/regex';
 
+const { pathToRegexp } = require('path-to-regexp');
```

Comment:
Can we not import this like previous?

Reply:
I've followed the method mentioned in the documentation, the developer suggests using this method and using the previous import method causes test failures.

Reply:
Adding curly brackets around the method being imported seems to fix the failing tests.